.github/workflows/run-tests.yml

```diff
@@ -20,10 +20,10 @@ jobs:
           - laravel: 10.*
             testbench: 8.*
           - laravel: 9.*
-            testbench: 7.*
+            testbench: ^7.22
         exclude:
-          - laravel: 10.*
-            php: 8.0
+          - php: 8.0
```

Comment:
Shouldn't this be like the following:
```exclude:
          - laravel: 10.*
            php: 8.0
```

Reply:
I guess no because here we said to exclude laravel 10 on PHP 8.0

Reply:
That's what it's supposed to do, right? 

Seems like this PR works, though: https://github.com/spatie/laravel-package-tools/pull/81

Reply:
Oh okay so I believe it does the same thing, look here: [ryangjchandler/blade-capture-directive#12](https://github.com/ryangjchandler/blade-capture-directive/pull/12/files#diff-7314d0ebbd2e9537ae4889316745b4fd2fa43cb86275c9caae18a86ba228b642R24-R26)